css/at-rules.json

```
@@ -120,6 +120,20 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@counter-style"
   },
+  "@container": {
+    "syntax": "@container <container-condition># {\n  <block-contents>\n}",
+    "interfaces": [
+      "CSSGroupingRule",
+      "CSSConditionRule",
+      "CSSContainerRule"
+    ],
+    "groups": [
+      "CSS Conditional Rules",
+      "CSS Containment"
+    ],
```

```suggestion
    "groups": [
      "CSS Conditional Rules"
    ],
```

since it is only defined in [CSS Conditional Rules](https://drafts.csswg.org/css-conditional/), not in [CSS Containment](https://drafts.csswg.org/css-contain/)

only "CSS Conditional Rules" should be presented

---

> * `groups` (array of strings): CSS is organized in modules like "CSS Fonts" or "CSS Animations". MDN organizes features in these groups as well — `groups` should contain the name of the module(s) the at-rule is defined in.

The docs for a reference.